doc/nrf/libraries/others/fatal_error.rst

```diff
@@ -0,0 +1,38 @@
+.. _lib_fatal_error:
+
+Fatal error handler
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |NCS| provides an implementation of the fatal error handler that overrides Zephyr's default fatal error handling implementation.
+
+Overview
+********
+
+The library defines the :c:func:`k_sys_fatal_error_handler()` function, which is declared in Zephyr's :ref:`fatal error handling API <zephyr:fatal>`.
+Its implementation in the |NCS| is standardized for most samples.
+
+Configuration
+*************
+
+When building for an embedded target, the default behavior of the :c:func:`k_sys_fatal_error_handler()` function in case of a fatal error is to reboot the application.
+You can modify the default behavior of the library not to reboot the application.
+To have the application enter an endless loop, change the :kconfig:`CONFIG_RESET_ON_FATAL_ERROR` Kconfig option to ``n``.
+
+Library files
+*************
+
+This library can be found under :file:`nrf/lib/fatal_error/` in the |NCS| folder structure.
```

Comment:
It would be good to not start a path from `nrf`. I think that it is not mandatory to have NCS under directory which name is nrf

Reply:
fixed